charts/internal/shoot-system-components/charts/rook-ceph/templates/toolbox.yaml

```diff
@@ -18,9 +18,16 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
       - name: rook-ceph-tools
-        image: rook/ceph:master
+        image: {{ index .Values.images "ceph-toolbox" }}
         command: ["/tini"]
         args: ["-g", "--", "/usr/local/bin/toolbox.sh"]
+        resources:
+          requests:
+            cpu: “1”
+            memory: “1Gi”
+          limits:
+            cpu: “2”
+            memory: “2Gi
```

Comment:
Same here; see above

Reply:
Ditto

Reply:
yeah way too much. you only check health or issue some admin commands through the toolbox. 100 millicores and 100MB should by far be enough

Reply:
Same